### DISCUSSION TITLE
How to use other model than Openai gpt

### DISCUSSION THREAD

**Discussion by lu-ph** [microsoft/autogen#4249] [Q&A] [answered]
How to use other models? Im in Hongkong and openai is not available. Im new for coding.

**Comment by jackgerrits** [comment, marked as answer]
You can find the model clients maintained by us [here](https://microsoft.github.io/autogen/dev/reference/python/autogen_ext/autogen_ext.models.html).

Since it is hard for us to maintain the diverse slate of model clients that people might want we are inviting people to create their own as [community extensions](https://microsoft.github.io/autogen/dev/user-guide/extensions-user-guide/index.html#creating-your-own-community-extension). Creating a custom model client is a matter of implementing [this interface](https://microsoft.github.io/autogen/dev/reference/python/autogen_core/autogen_core.components.models.html#autogen_core.components.models.ChatCompletionClient).

That being said, we hope to maintain a few more of the most popular ones in `autogen-ext` but just haven't gotten around to porting them from 0.2 yet.

**Reply by DarkShark-RAz** [reply]
the links open to a 404
